app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,7 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            
```

Comment:
Remove this line it's not necessary

Reply:
I just follow the convention from the other PR corrections: the xml tags that have childs have and empty line above and below. For example: \<application> and \<manifest> has both lines.

But I can even remove the line if you want.